lib/php/rendition-factory/src/DTO/BuildConfig/Transformation.php

```diff
@@ -25,4 +26,19 @@ public function getOptions(): array
     {
         return $this->options;
     }
+
+    public function isEnabled(): bool
+    {
+        return $this->enabled;
+    }
+
+    public function asArray(): array
```

Comment:
`toArray` is widely used

```suggestion
    public function toArray(): array
```